static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-articles-viewed.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import {
+    makeEpicABTest,
+    defaultButtonTemplate,
+    buildEpicCopy,
+} from 'common/modules/commercial/contributions-utilities';
+import { getArticleViewCount } from 'common/modules/onward/history';
+import { getSync as geolocationGetSync } from 'lib/geolocation';
+
+// Use must have read at least 5 articles in last 14 days
+const minArticleViews = 5;
+const articleCountDays = 14;
+
+const articleViewCount = getArticleViewCount(articleCountDays);
+
+const highlightedText =
+    'Support The Guardian from as little as %%CURRENCY_SYMBOL%%1 – and it only takes a minute. Thank you.';
+
+const USUKControlCopy = {
+    heading: 'Since you’re here…',
+    paragraphs:
+        '… we have a small favour to ask. More people are reading and supporting The Guardian’s independent, investigative journalism than ever before. And unlike many new organisations, we have chosen an approach that allows us to keep our journalism accessible to all, regardless of where they live or what they can afford. But we need your ongoing support to keep working as we do.\n' +
+        'The Guardian will engage with the most critical issues of our time – from the escalating climate catastrophe to widespread inequality to the influence of big tech on our lives. At a time when factual information is a necessity, we believe that each of us, around the world, deserves access to accurate reporting with integrity at its heart.\n' +
+        'Our editorial independence means we set our own agenda and voice our own opinions. Guardian journalism is free from commercial and political bias and not influenced by billionaire owners or shareholders. This means we can give a voice to those less heard, explore where others turn away, and rigorously challenge those in power.\n' +
+        'We need your support to keep delivering quality journalism, to maintain our openness and to protect our precious independence. Every reader contribution, big or small, is so valuable. \n',
+    highlightedText,
+};
+
+const ROWControlCopy = {
+    heading: 'More people in %%COUNTRY_NAME%%...',
+    paragraphs:
+        '... like you, are reading and supporting The Guardian’s independent, investigative journalism than ever before. And unlike many new organisations, we have chosen an approach that allows us to keep our journalism accessible to all, regardless of where they live or what they can afford. But we need your ongoing support to keep working as we do.\n' +
```

Comment:
the ellipsis in this control has spaces, but the one in the USUKControl does not - not sure which is correct but I'm guessing it's not both

Reply:
copied from the doc, @ionamckendrick ?

Reply:
Both should have a space after the ellipses

Reply:
Our format for Epics is always: 
**Hook...**
**... I'm the first sentence**

So, no space in the hook; yes space in the opening para.

Reply:
Sorry, I misunderstood. You mean in terms of how close the ellipses are. It should always be the version you see here in the country variant, please.